tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php

```diff
@@ -631,4 +631,19 @@ public function inheritdocAnnotation($a)
 		return true;
 	}
 
+	/**
+	 * @inheritDoc
```

Comment:
@andrew-demb I'm sorry, i missed this in first review. There should be `{@inheritDoc}`

Reply:
Done.